tests/neg/LiquidClass1.hs

```diff
@@ -0,0 +1,39 @@
+module LiquidClass where
+
+
+-- | Typing classes
+-- | Step 1: Refine type dictionaries:
+
+class Compare a where
+	cmax :: a -> a -> a
+	cmin :: a -> a -> a
+
+instance Compare Int where	
+{-@ instance Compare Int where 
+	cmax :: Odd -> Odd -> Odd ;
```

Comment:
What is the alternative? 
We need to know where the instance types stop. Indentation would be a solution..... but it seems quite complicated